tests/sentry/models/test_actor.py

```diff
@@ -0,0 +1,19 @@
+from sentry.models import Actor, User
+from sentry.testutils import TestCase
+
+
+class ActorTest(TestCase):
+    def test_pre_save(self):
+        user = self.create_user(email="test@pre_save.com")
+        assert user.actor is not None
```

Comment:
Could you verify that actor is the correct type?

Reply:
Yeah. But wouldn't the database insertion fail if it wasn't a FK to actor?

Reply:
I actually meant `Actor.type` is correct, words are hard